src/tags.py

```diff
@@ -393,40 +349,34 @@ async def remove_id(itx: Interaction, tag_id: int) -> None:
         args = (tag_id, itx.user.id)
         clause = "rowid = ? AND ownerID = ?"
 
-    query = f"DELETE FROM tags WHERE {clause} RETURNING rowid, name"
+    query = f"DELETE FROM tags WHERE {clause} RETURNING name"
     async with itx.client.pool.acquire() as conn, conn.transaction():
-        deleted_info = await conn.fetchone(query, args)
+        name, = await conn.fetchone(query, args) or ("",)
 
-    if deleted_info is None:
+    if name:
         return await itx.response.send_message(
-            f"You do not own a tag with ID {tag_id}."
+            f"Deleted tag named {name!r} (ID {tag_id})."
         )
 
-    rowid, name = deleted_info
     await itx.response.send_message(
-        f"Deleted tag named {name!r} (ID {rowid})."
+        f"You do not own a tag with ID {tag_id}."
     )
 
 
-async def _send_tag_info(
-    itx: Interaction,
-    tag_name: str,
-    row: sqlite3.Row
-) -> None:
-    """Expects row in this format: rowid, uses, ownerID, time_created"""
+async def _send_tag_info(itx: Interaction, row: sqlite3.Row) -> None:
+    """Expects row in this format: name, rowid, uses, ownerID, time_created"""
 
     embed = discord.Embed(colour=discord.Colour.blurple())
     embed.set_footer(text="Tag created")
 
-    rowid, uses, owner_id, time_created = row
-
-    embed.title = tag_name
-    embed.timestamp = datetime.fromtimestamp(time_created, tz=timezone.utc)
+    # Unpacking the row
+    embed.title, rowid, uses, owner_id, time_created = row
 
     user = (
         itx.client.get_user(owner_id)
         or (await itx.client.fetch_user(owner_id))
     )
```

Comment:
Suggestion: make a get or fetch method within the subclassed bot and use that.

It would look much cleaner and can be reused in the leaderboards too when that arrives